gpcontrib/arenadata_toolkit/arenadata_toolkit--1.3--1.4.sql

```diff
@@ -0,0 +1,375 @@
+/* gpcontrib/arenadata_toolkit/arenadata_toolkit--1.3--1.4.sql */
+
+DO $$
+BEGIN
+	IF (SELECT a.attrelid IS NULL
+		FROM pg_class c
+		JOIN pg_namespace n ON n.oid = c.relnamespace AND
+		                       n.nspname = 'arenadata_toolkit'
```

Comment:
Typically, a condition, that is not related to both tables at the same time, is moved to the where block:
```sql
	IF (SELECT a.attrelid IS NULL
		FROM pg_class c
		JOIN pg_namespace n ON n.oid = c.relnamespace
		LEFT JOIN pg_attribute a ON a.attrelid = c.oid AND a.attname = 'tablespace_location'
		WHERE c.relname = 'db_files_current' AND n.nspname = 'arenadata_toolkit')
	THEN
```